docs/Colab version control.md

```diff
@@ -0,0 +1,22 @@
+**Colab Version Control**
+
+In line with best practices, developers using Google colaboratory to write their scripts are encouraged to integrate their notebooks into Github for version control.
+
+**Steps to integrate Colab with Github**
+1. Either clone the nsmq repository and create a local branch or create a remote branch directly in Github. All branches must have the right naming conventions.
```

Comment:
let's go with create a local branch. Cloning or creating a remote branch will make it difficult for folks to easily add in subsequent changes after the PR is created in case there are things to be updated in the review etc

Reply:
disregard the comment on cloning. Wrongly thought it was forking a branch